db/migrate/20221102031315_add_users_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :users, null: false, foreign_key: true
```

Comment:
* According to the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_1.md#:~:text=Table%20and%20column%20names%20should%20match%20the%20ERD%20diagram.) `Table and column names should match the ERD diagram.` , Kindly ensure your foreign keys are the same as in the ERD diagram currently yours says `users_id` instead of `author_id` 
![image](https://user-images.githubusercontent.com/52497246/199391573-c31baa01-d157-48e1-b02c-f06e2d2d2378.png)